### PR TITLE
logger added and format changes

### PR DESCRIPTION
**Pull request by c-kodman**

### CODE REVIEW

File: ccmlib/dse_cluster.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def load_credentials_from_file(self, dse_credentials_file):
                 if parser.has_option('dse_credentials', 'dse_password'):
                     self.dse_password = parser.get('dse_credentials', 'dse_password')
             else:
-                print_("Warning: {} does not contain a 'dse_credentials' section.".format(dse_credentials_file), file=sys.stderr)
+                common.warning("Warning: {} does not contain a 'dse_credentials' section.".format(dse_credentials_file), file=sys.stderr)
```
Comment (ptnapoleon) [inline ccmlib/dse_cluster.py]: `common.warning` doesn't take a file kwarg

File: ccmlib/node.py [hunk 2]
```diff
@@ -1798,7 +1798,7 @@ def resume(self):
             p.resume()
         except ImportError:
             if common.is_win():
-                print_("WARN: psutil not installed. Resume functionality will not work properly on Windows.")
+                common.warning("WARN: psutil not installed. Resume functionality will not work properly on Windows.")
```
Comment (ptnapoleon) [inline ccmlib/node.py]: Now that we are logging, we no longer want the log level prepended in the message, as that will happen automatically now. Meaning these will now be redundant

File: ccmlib/node.py [hunk 3]
```diff
@@ -275,21 +275,21 @@ def show(self, only_status=False, show_cluster=True):
...
-            print_("%s%s=%s" % (indent, 'jmx_port', self.jmx_port))
-            print_("%s%s=%s" % (indent, 'remote_debug_port', self.remote_debug_port))
-            print_("%s%s=%s" % (indent, 'byteman_port', self.byteman_port))
-            print_("%s%s=%s" % (indent, 'initial_token', self.initial_token))
+                print_("{}{}={}".format(indent, 'binary', self.network_interfaces['binary']))
+            print_("{}{}={}".format(indent, 'storage', self.network_interfaces['storage']))
+            print_("{}{}={}".format(indent, 'jmx_port', self.jmx_port))
+            print_("{}{}={}".format(indent, 'remote_debug_port', self.remote_debug_port))
+            print_("{}{}={}".format(indent, 'byteman_port', self.byteman_port))
+            print_("{}{}={}".format(indent, 'initial_token', self.initial_token))
```
Comment (ptnapoleon) [inline ccmlib/node.py]: If @mambocab were here, he'd suggest a clever way of reducing all this code dupe, which I won't force on you. But I ping him here to see if he agrees that I should do it

Comment (mambocab) [inline ccmlib/node.py]: Eh, not worth the churn right now. I don't understand why this code still uses `print_` though.

Comment (ptnapoleon) [inline ccmlib/node.py]: What else would it use

**Comment by ptnapoleon** [comment]
Can you run dtest against these changes and lmk what happens?

**Comment by mshuler** [comment]
I was attempting to help with testing this branch with dtest and we sort of foobar'ed a rebase, as you can see from the odd commit history. I pulled @c-kodman 's branch in a new remote and created a squashed commit off master commit fb3cfa7 here:
https://github.com/mshuler/ccm/tree/pr514

I have a dtest running on my pr514 ccm branch at:
http://cassci.datastax.com/view/Dev/view/mshuler/job/mshuler-trunk_ccm-pr514-dtest/1/

-- Michael

**Comment by mshuler** [comment]
That job ended with gusto! 868 failures with lots of:
```
Error Message

unexpected indent (repository.py, line 161)
Stacktrace

Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/usr/local/lib/python2.7/dist-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/local/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/local/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/automaton/cassandra-dtest/paging_test.py", line 10, in <module>
    from assertions import assert_invalid
  File "/home/automaton/cassandra-dtest/assertions.py", line 8, in <module>
    from tools import rows_to_list
  File "/home/automaton/cassandra-dtest/tools.py", line 17, in <module>
    from ccmlib.node import Node
  File "/home/automaton/ccm/ccmlib/node.py", line 23, in <module>
    from ccmlib.repository import setup
  File "/home/automaton/ccm/ccmlib/repository.py", line 161
    common.info("Checking out requested {} ({})".format(branch_or_sha_tag, git_branch))
    ^
IndentationError: unexpected indent
```

**Comment by mshuler** [comment]
Unindented repository.py, line 161 and trying another run from my pr514 branch:
http://cassci.datastax.com/job/mshuler-trunk_ccm-pr514-dtest/2/

**Comment by mshuler** [comment]
Far fewer errors, but there are a few of these:
```
Error Message

Building C* version git:cassandra-2.1 failed. Attempted to delete /home/automaton/.ccm/repository/gitCOLONcassandra-2.1 but failed. This will need to be manually deleted
-------------------- >> begin captured logging << --------------------
dtest: DEBUG: cluster ccm directory: /mnt/tmp/dtest-WALrXa
dtest: DEBUG: Custom init_config not found. Setting defaults.
dtest: DEBUG: Done setting configuration options:
{   'authenticator': 'PasswordAuthenticator',
    'authorizer': 'CassandraAuthorizer',
    'initial_token': None,
    'num_tokens': '32',
    'phi_convict_threshold': 5}
--------------------- >> end captured logging << ---------------------
Stacktrace

  File "/usr/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/home/automaton/cassandra-dtest/upgrade_internal_auth_test.py", line 37, in upgrade_to_22_test
    self.do_upgrade_with_internal_auth("git:cassandra-2.2")
  File "/home/automaton/cassandra-dtest/upgrade_internal_auth_test.py", line 60, in do_upgrade_with_internal_auth
    cluster.set_install_dir(version="git:cassandra-2.1")
  File "/home/automaton/ccm/ccmlib/cluster.py", line 93, in set_install_dir
    dir, v = repository.setup(version, verbose)
  File "/home/automaton/ccm/ccmlib/repository.py", line 37, in setup
    clone_development(GIT_REPO, version, verbose=verbose)
  File "/home/automaton/ccm/ccmlib/repository.py", line 198, in clone_development
    raise CCMError("Building C* version %s failed. Attempted to delete %s but failed. This will need to be manually deleted" % (version, target_dir))
"Building C* version git:cassandra-2.1 failed. Attempted to delete /home/automaton/.ccm/repository/gitCOLONcassandra-2.1 but failed. This will need to be manually deleted\n-------------------- >> begin captured logging << --------------------\ndtest: DEBUG: cluster ccm directory: /mnt/tmp/dtest-WALrXa\ndtest: DEBUG: Custom init_config not found. Setting defaults.\ndtest: DEBUG: Done setting configuration options:\n{   'authenticator': 'PasswordAuthenticator',\n    'authorizer': 'CassandraAuthorizer',\n    'initial_token': None,\n    'num_tokens': '32',\n    'phi_convict_threshold': 5}\n--------------------- >> end captured logging << ---------------------"
Standard Output

http://git-wip-us.apache.org/repos/asf/cassandra.git git:cassandra-2.1
```

and lots of this error:
```
Error Message

global name 'common' is not defined
-------------------- >> begin captured logging << --------------------
dtest: DEBUG: cluster ccm directory: /mnt/tmp/dtest-QOFPgr
dtest: DEBUG: Custom init_config not found. Setting defaults.
dtest: DEBUG: Done setting configuration options:
{   'initial_token': None,
    'num_tokens': '32',
    'phi_convict_threshold': 5,
    'range_request_timeout_in_ms': 10000,
    'read_request_timeout_in_ms': 10000,
    'request_timeout_in_ms': 10000,
    'truncate_request_timeout_in_ms': 10000,
    'write_request_timeout_in_ms': 10000}
--------------------- >> end captured logging << ---------------------
Stacktrace

  File "/usr/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/home/automaton/cassandra-dtest/upgrade_tests/storage_engine_upgrade_test.py", line 255, in upgrade_with_wide_partition_reversed_test
    self.upgrade_with_wide_partition(query_modifier=" ORDER BY t DESC")
  File "/home/automaton/cassandra-dtest/upgrade_tests/storage_engine_upgrade_test.py", line 260, in upgrade_with_wide_partition
    session = self._setup_cluster()
  File "/home/automaton/cassandra-dtest/upgrade_tests/storage_engine_upgrade_test.py", line 27, in _setup_cluster
    cluster.set_install_dir(version="2.1.9")
  File "/home/automaton/ccm/ccmlib/cluster.py", line 93, in set_install_dir
    dir, v = repository.setup(version, verbose)
  File "/home/automaton/ccm/ccmlib/repository.py", line 75, in setup
    common.warning("Downloading {} failed, due to {}. Trying to build from git instead.".format(version, e))
"global name 'common' is not defined\n-------------------- >> begin captured logging << --------------------\ndtest: DEBUG: cluster ccm directory: /mnt/tmp/dtest-QOFPgr\ndtest: DEBUG: Custom init_config not found. Setting defaults.\ndtest: DEBUG: Done setting configuration options:\n{   'initial_token': None,\n    'num_tokens': '32',\n    'phi_convict_threshold': 5,\n    'range_request_timeout_in_ms': 10000,\n    'read_request_timeout_in_ms': 10000,\n    'request_timeout_in_ms': 10000,\n    'truncate_request_timeout_in_ms': 10000,\n    'write_request_timeout_in_ms': 10000}\n--------------------- >> end captured logging << ---------------------"
```

**Comment by c-kodman** [comment]
https://cassci.datastax.com/view/Dev/view/c-kodman/job/c-kodman-trunk-dtest/6/testReport/

**Comment by ptnapoleon** [comment]
+1

File: ccmlib/node.py (hunk 3 above)
Comment (mambocab) [inline ccmlib/node.py]: logging?

Comment (mambocab) [inline ccmlib/node.py]: Oh or is this command output? lol

**Pull request merged**